src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/CombinationController.php

```diff
@@ -288,6 +289,36 @@ public function deleteAction(int $combinationId): JsonResponse
         ]);
     }
 
+    /**
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))")
+     *
+     * @param int $productId
+     * @param Request $request
+     *
+     * @return JsonResponse
+     */
+    public function bulkDeleteAction(int $productId, Request $request): JsonResponse
+    {
+        $combinationIds = $request->request->get('combinationIds');
+        if (!$combinationIds) {
```

Comment:
```suggestion
        if (!Validate::isArrayWithIds($combinationIds)) {
```

Reply:
no need for that. BulkDeleteCombinationCommand asserts ids in constructor (and converts themto CombinationId[] value objects

Reply:
@zuk3975 because you suppose that **combinationIds** is come from **BulkDeleteCombinationCommand**

Reply:
sorry I don't understand what are you saying :smile:

Reply:
I think what @zuk3975 is saying is that even without this check when the parameter is used to create `BulkDeleteCombinationCommand` (which happens 7 lines lower) it will inevitably be validated so the check here is redundant and would bring less information as the command exception

The purpose of the test here is simply to ensure the presence of the mandatory parameter, not its full validity